api/migrations/versions/c79ebe42885e_add_parent_id_to_response_v2.py

```diff
@@ -0,0 +1,25 @@
+"""
```

Comment:
Then once you've done this, we need to get ahead of one more issue. There have been migrations applied to the db since you opened this PR which we need to handle. So merge in the latest changes from the `develop` branch, then run the below command and push the additional migration that's created. 

```bash
# create merge migration
docker-compose run web python ./api/run.py db merge --directory api/migrations heads

# apply it to local db
docker-compose run web python ./api/run.py db upgrade --directory api/migrations
```

See the [docs](https://github.com/deep-learning-indaba/Baobab/tree/develop/api#merging-migrations) for more details on this if you're curious.